generators/app/prompts.js

```diff
@@ -9,6 +13,21 @@ module.exports = {
           name: 'seedbank',
           checked: true,
           value: "gem 'seedbank'"
+        },
+        {
+          name: 'rack-cors',
+          checked: true,
+          value: "gem 'rack-cors'"
+        },
+        {
+          name: 'redis',
+          checked: true,
+          value: "gem 'redis', '~> 3.0'"
+        },
+        {
+          name: 'bcrypt',
+          checked: true,
+          value: "gem 'bcrypt', '~> 3.1.7'"
```

Comment:
More on that: I choose to let `minitest` default because I think `rspec` and `minitest` will get the job done and will be harder to build and maintain two versions of template. I think that maybe this should be a standard, will make easier to switch between projects.